back-end/api/quickstart/tests.py

```diff
@@ -27,3 +29,46 @@ def test_get_valid_author(self):
   def test_get_invalid_author(self):
     response = client.get('/authors/invalidId/')
     self.assertEqual(response.status_code, status.HTTP_404_NOT_FOUND)
+
+
+class GetPostById(TestCase):
+
+  def setUp(self):
+    self.john = Author.objects.create(
+      _id='testId',
+      displayName='John Doe',
+      url="testUrl",
+      github="testGithub"
+    )
+
+    self.post = Post.objects.create(
+      _id='123',
+      title='testpost',
+      description='i am a test post',
+      source='source post id',
+      origin='origin post id',
+      visibility='Public',
+      unlisted='True',
+      isPrivateToFriends='True',
+      author=AuthorSerializer(self.john).data,
```

Comment:
If you implement the changes I talked about above, the test might get a bit difficult (you'd have to do author=testId here, but then it's not so easy to make assertions on the HTTP response anymore). Maybe we leave it for a future PR?

Reply:
The assertions will pass now, just later on when we have to actually return the json object will these test not pass but that can be dealt then.